contracts/client/ClientFrontend.sol

```diff
@@ -0,0 +1,97 @@
+pragma solidity ^0.5.15;
+pragma experimental ABIEncoderV2;
+
+import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
+import { Tx } from "../libs/Tx.sol";
+import { Types } from "../libs/Types.sol";
+import { Transition } from "../libs/Transition.sol";
+import { Offchain } from "./Offchain.sol";
+
+contract ClientFrontend {
+    using SafeMath for uint256;
+    using Tx for bytes;
+    using Types for Types.UserState;
+
+    function decodeTransfer(bytes calldata txBytes)
```

Comment:
all the variables named txBytes are encodedTxBytes correct? A small nitpick would be to change the variable name to reflect that, but you can ignore that if you don't feel a need for it

Reply:
Yes, they are encoded txBytes. I can rename those